langkit/templates/pkg_introspection_body_ada.mako

```diff
@@ -9,11 +9,38 @@ with Ada.Strings.Unbounded.Hash;
 with ${ada_lib_name}.Implementation;
```

Comment:
General comment: this commit seems to be a bit doc-lacking. There are both undocumented types and IMHO the code generation part could be a bit more documented.

Reply:
After live discussion: this note was written thinking this patch hunk was for a package specification (which contains most documentation). This is the body, so the API is not documented here on purpose, but internals should be discribed as needed. We’ll add more comments if necessary.